protobuf/v1/pool.proto

```diff
@@ -59,6 +65,11 @@ message DestroyPoolRequest {
   string uuid = 1;  // uuid of the pool
 }
 
+// Export pool arguments.
+message ExportPoolRequest {
```

Comment:
Wonder if doing this by name with optional (and strongly recommended) UUID.

Reply:
Make sense and the same can be done in DeletePoolRequesr as well.